Formula/glibc.rb

```diff
@@ -190,6 +239,22 @@ def post_install
     share.install_symlink sys_zoneinfo if sys_zoneinfo.exist? && !brew_zoneinfo.exist?
   end
 
+  def caveats
+    <<~EOS
+      The Homebrew's Glibc has been installed with the following executables in your PATH:
+        #{HOMEBREW_PREFIX}/bin/ldd
+        #{HOMEBREW_PREFIX}/bin/ld.so
+        #{HOMEBREW_PREFIX}/sbin/ldconfig
```

Comment:
Let's make this keg-only.

Reply:
> Let's make this keg-only.

Does this mean the brewed `ld.so` will not be present in the user's `PATH` (`HOMEBREW_PREFIX/bin`)?